ReSharper.FSharp/src/FSharp.Psi/src/Impl/DeclaredElement/FSharpPropertyBase.cs

```diff
@@ -18,17 +17,23 @@ internal abstract class FSharpPropertyMemberBase<TDeclaration> : FSharpPropertyB
     {
       var prop = GetProperty(mfv);
 
-      IsReadable = prop.HasGetterMethod || prop.IsPropertyGetterMethod ||
-                   prop.IsModuleValueOrMember && !prop.IsMember;
+      var isReadable = IsReadable = prop.HasGetterMethod || prop.IsPropertyGetterMethod ||
+                                    prop.IsModuleValueOrMember && !prop.IsMember;
 
-      IsWritable = prop.IsMutable || prop.HasSetterMethod || prop.IsPropertySetterMethod;
+      var isWritable = IsWritable = prop.IsMutable || prop.HasSetterMethod || prop.IsPropertySetterMethod;
+
+      Getter = isReadable ? new FSharpPropertyAccessor(prop.GetterMethod, this) : null;
```

Comment:
Do we really need to store it in all cases? Wouldn't the implicit one be sufficient in the most cases?